math/overflow_test.go

```diff
@@ -17,8 +17,10 @@ func TestAddUint64(t *testing.T) {
 	c := uint64(3)
 	sum, err = AddUint64WithErr(b, c)
 	require.Equal(t, ErrAdditionOverflow, err)
+	require.Equal(t, b+c, sum)
```

Comment:
Why should `sum` be equal to `b + c` (the normal golang sum) after `AddUint64WithErr` has already returned an overflow error?

Reply:
this is what the function returns.

```
func AddUint64WithErr(a, b uint64) (uint64, error) {
	s := a + b
	if s >= a && s >= b {
		return s, nil
	}

	return s, ErrAdditionOverflow
}
```